index.md

```diff
@@ -10,18 +10,57 @@ layout: standard-page
 Good humored, cooperative, competitive, consensual torture, for the purposes of
 exploring strategies, design, progress and history of gaming
 
-### Members
-* There is no single leader of the Zargon community. Zargon is a community run
-  effort. All members are encouraged to participate in this effort.
-* You may invite new members to join in gaming, but they are your
-  responsibility.
-* New invitees should come to provisional games (e.g., Friday games, or similar
-  virtual games) for the sake of the invitee and the community.
-* If community members at large do not like a guest then that guest may not be
-  invited to future events.
+## Membership
+There is no single leader of the Zargon community. Zargon is a community run
+effort. All members are encouraged to participate in this effort.
+
+The following guidelines are intended to allow us to maintain an inclusive
+environment where all members feel comfortable gaming with any other member of
+the group rather than fracturing in cliques. They are enumerated in detail so
+as to mitigate subjectivity in role assignment.
+
+### Obligations
 * Members are expected to maintain basic personal hygiene.
 * Members are expected to be gracious and respect additional restrictions of
-  venue (e.g. no smoking inside; no young children; allergy considerations).
+  venue
+    - e.g. no smoking inside; no young children; allergy considerations.
+
+### Active Members
+* An active member is one who regularly does one or more of the following
+  - Attends scheduled gaming or social events
+  - Paricipates in virtual conversations
+* Active members are noted by the **Zargonaut** role in the group discord.
+
+### Contributing Members
+* A contributing member is one who regularly does one or more of the following
+  - Schedules game sessions
+  - Schedules social events
+  - Brings new games for the group to try
+  - Teaches games at gaming events
+  - Creates content to aid the group (e.g. rules handouts, character
+    illustrations, game storage solutions)
+* Contributing members are noted by the **Adept** role in the group discord.
```

Comment:
**The Council** was also a proposed role for this as well, did we want to consider that?

We should include a process for ascending to the rank of **Adept** / **The Council** as well. Maybe a petition from a **Zargonaut** and a majority vote by other contributing members?